src/components/localtunneldebugger/localtunneldebugger.ts

```diff
@@ -0,0 +1,46 @@
+import * as vscode from 'vscode';
+
+import { LocalTunnelDebugProvider } from './localtunneldebugger.extension';
+import { getLocalTunnelDebugProvider } from '../config/config';
+
+export class LocalTunnelDebugger {
+    private readonly providers = Array.of<LocalTunnelDebugProvider>();
+
+    register(provider: LocalTunnelDebugProvider): void {
+        console.log(`Registered local tunnel debugger type ${provider.id}`);
+        this.providers.push(provider);
+    }
+
+    startLocalTunnelDebugProvider(target?: any): void {
+        const browseExtensions = "Find Providers on Marketplace";
+        if (this.providers.length === 0)
+        {
```

Comment:
Nit: could you please put this bracket on the above line, to respect the convention in the rest of the file?

Reply:
Thanks!